internal/verifier/verifier_test.go

```diff
@@ -93,6 +93,48 @@ func TestReproducibleProvenanceVerifier_badCommand(t *testing.T) {
 	}
 }
 
+func TestLoadAndVerifyProvenances_HasBuildCmd(t *testing.T) {
```

Comment:
Perhaps add a test for `ProvenanceIRVerifier.Verify()` where `Want` contains an empty array for `BuildCmds`. I am not sure what we'd expect in that case.

Reply:
Again, my vision with #166 would be. If we have an empty array in `Want`, the `ProvenanceIRVerifier` does not verify that value.

Reply:
Will there be a way to verify that a command does not exist in the provenance?

Reply:
What exactly would we want to check? 

I would have thought, whether a command exists depends on the format of the provenance, e.g., for SLSA v.02, `buildCmd` does not exist. So I am not sure what we can check.

The way I imagined we can check on whether the value should exists for a provenance format is through reference values (#166 ).

Reply:
Actually I think a valid provenance (amber or slsav1) should always have a command. If the only thing we want to check about the build command is its presence, and not its exact value, then we probably don't even need a reference value for it.

The only thing we should consider in checking `hasBuildCommand` is the provenance version. In particular, in slsa02  we don't set the build command.

Reply:
> I would have thought, whether a command exists depends on the format of the provenance, e.g., for SLSA v.02, buildCmd does not exist. So I am not sure what we can check.

True. So I don't think a reference value is very helpful in this case. Unless we want to check the exact value of the build command. 

> The way I imagined we can check on whether the value should exists for a provenance format is through reference values (https://github.com/project-oak/transparent-release/issues/166 ).

I think reference values are one source of information, but the provenance format/version should be taken into account as well. Is there going to be a reference value for the accepted provenance format/version (e.g., slsa02, slsa1)?

Reply:
I added a draft PR to show how I thought the check on provenanceFormat/version (version is a better work) could work [#167](https://github.com/project-oak/transparent-release/pull/167).

NB: Sadly the changes are a bit hidden, because I need to base on this PR. I can submit this PR and we discuss on the next PR.

Reply:
I think it is better to add a test for `ProvenanceIRVerifier.Verify()` where `Want` contains an empty array for `BuildCmds`, and merge this PR, and continue the discussion on #167.

Reply:
Done.